pkg/codegen/internal/test/testdata/plain-object-disable-defaults/go/example/funcWithAllOptionalInputs.go

```diff
@@ -0,0 +1,74 @@
+// *** WARNING: this file was generated by test. ***
+// *** Do not edit by hand unless you're certain you know what you are doing! ***
+
+package example
+
+import (
+	"context"
+	"reflect"
+
+	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
+)
+
+// Check codegen of functions with all optional inputs.
+func FuncWithAllOptionalInputs(ctx *pulumi.Context, args *FuncWithAllOptionalInputsArgs, opts ...pulumi.InvokeOption) (*FuncWithAllOptionalInputsResult, error) {
+	var rv FuncWithAllOptionalInputsResult
+	err := ctx.Invoke("mypkg::funcWithAllOptionalInputs", args, &rv, opts...)
+	if err != nil {
+		return nil, err
+	}
+	return &rv, nil
+}
+
+type FuncWithAllOptionalInputsArgs struct {
+	// Property A
+	A *HelmReleaseSettings `pulumi:"a"`
+	// Property B
+	B *string `pulumi:"b"`
+}
+
+type FuncWithAllOptionalInputsResult struct {
+	R string `pulumi:"r"`
+}
+
+func FuncWithAllOptionalInputsOutput(ctx *pulumi.Context, args FuncWithAllOptionalInputsOutputArgs, opts ...pulumi.InvokeOption) FuncWithAllOptionalInputsResultOutput {
```

Comment:
There's no defaults here correct? Defaults don't apply to Invokes? Is this by design or deferred for later? Why is  funcWithAllOptionalInputs included in this test suite? I remember authoring it to test the Output version generation.